### ISSUE TITLE
No validator to check type of file uploaded in import project

### ISSUE THREAD

**Issue by kushthedude** [OpenRefine/OpenRefine#2386]
**Describe the bug**

No validator to check type of file uploaded in import project while importing from local computer

**To Reproduce**

1. Go to 'http://127.0.0.1:3333'
2. Click on 'Get data from this computer'
3. Select JPG, PNG or any other invalid format

**Expected behavior**

There should be an error of file type not supported

**Screenshots**

<img width="1202" alt="Screenshot 2020-03-11 at 1 59 46 AM" src="https://user-images.githubusercontent.com/44091822/76356489-0a498780-633c-11ea-9024-bb3cfaf07660.png">

**Comment by kushthedude** [comment]
@wetned @thadguidry If this is a valid issue, where can I find the list of formats we would like to support ?

**Comment by thadguidry** [comment]
We have importers for some file types.  But we don't limit users ability to use OpenRefine on any file that they have.  This way OpenRefine can be used to make sense of the file and structure and clean it if they wish.

**Comment by kushthedude** [comment]
What good the other file types are if the OpenRefine is set to work on some
specific file formats?

**Comment by thadguidry** [comment]
Sorry, I don't understand.  Are you asking why OpenRefine only has some importers and not 1000 ?

**Comment by thadguidry** [comment]
The concepts of OpenRefine are not unlike ETL programs.  Here we are talking about the E (extract) https://en.wikipedia.org/wiki/Extract,_transform,_load

What I am trying to explain is that OpenRefine allows any flat file to be extracted.  If the file has some linefeeds, then line-based importer will be detected, but if it has no lines, a user can still preview the file and might knowing understand the file structure and use some other byte sequence to create lines or they may do this outside of OpenRefine and then import as line-based...without OpenRefine even understand the structure or file format.

In other words, we don't want to limit to only a few recently understood file formats, but potentially any file that a user might want to work with in OpenRefine, if they want to.  This is especially important in GLAM domains where sequenced data streams where written to tape and are partitioned as individual sequenced flat files, then an ETL program is run to Extract, Transform, and Load the data into newer databases.

**Comment by kushthedude** [comment]
>  Many choose to use OpenRefine for handling their Extract, Transform, and Load processing because it readily allows humans to interactively preview their data...unlike other ETL programs that don't allow much interactivity. (Apache NiFi, etc.) 

Got your point, thanks for the awesome explanation yet again 👍
